### PR TITLE
BE | Ask VA API: Minor bug fix and update to inquiry_params

### PR DESCRIPTION
**Pull request by Khoa-V-Nguyen**
## Summary

- Minor bug fix for the mapping
- updated `inquiry_params` in `shared_helper.rb`

## Related issue(s)

- https://github.com/department-of-veterans-affairs/ask-va/issues/1374

## Testing done

- [x] *New code is covered by unit tests*

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [x]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x]  Feature/bug has a monitor built into Datadog (if applicable)
- [x]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected

### CODE REVIEW

**Review by hemeshvpatel — APPROVED** [review]
Lgtm!

File: modules/ask_va_api/app/lib/ask_va_api/inquiries/payload_builder/shared_helpers.rb [hunk 1]
```diff
@@ -5,26 +5,32 @@ module Inquiries
...
+          pronouns[:pronouns_not_listed_text].presence || pronouns.key(true)&.to_s&.tr('_', '/')
         end
 
         def contact_field(field, inquiry_details, inquiry_params)
           inquiry_details[:level_of_authentication] == 'Business' ? inquiry_params[field] : nil
         end
 
         def fetch_state_code(state)
-          I18n.t('ask_va_api.states').invert[state].to_s
+          return if state.nil?
```
Comment (rmtolmach) [inline modules/ask_va_api/app/lib/ask_va_api/inquiries/payload_builder/shared_helpers.rb:31]: Are there any tests for all of the `nil` returns happening, including this one?

Comment (Khoa-V-Nguyen) [reply modules/ask_va_api/app/lib/ask_va_api/inquiries/payload_builder/shared_helpers.rb:31]: Yes, it's in the shared_context.rb

Screenshot 2024-10-28 at 3 25 14 PM

Comment (rmtolmach) [reply modules/ask_va_api/app/lib/ask_va_api/inquiries/payload_builder/shared_helpers.rb:31]: ok, thanks.

**Review by rmtolmach — APPROVED** [review]

**Pull request merged**